lib/logement.ts

```diff
@@ -60,20 +60,19 @@ function isOwner(_logementType) {
   return _logementType === "proprietaire"
 }
 
-function captureCharges(_logementType, _locationType) {
-  return !(Logement.isOwner(_logementType) || _locationType === "meuble")
+function captureCharges(_logementType) {
+  return _logementType === "locataire"
```

Comment:
J'ai l'impression que ça change la logique.
Cette partie du code a beaucoup d'enum sous forme de chaînes de caractères. Ça pourrait être un lieu d'expérimentation des enums en TS.

```suggestion
function captureCharges(_locationType) {
  return _locationType === "vide" || _locationType === "foyer"
```